CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### 1.3.2
+
```

Comment:
I think https://github.com/sendsmaily/smaily-opencart-module/issues/72 could be also added to 1.3.2 version release but merged into the master branch with a separate pull-request.

Reply:
OK, but then this PR shouldn't bump the version.

Reply:
We need then to track what has been done and when to release on `project` page. Or add something like this to changelog:
### 1.3.2
- Reset abandoned cart sent status when customer empties cart
- TODO: #72

It's hard to keep track of what is done for what version of OpenCart and what hotfix/feature should be done next. I can remove the version bump and the changelog message, but then there is no track than pull-request merge that this feature was implemented in main branch. Kind of hides a feature that has been already done.

The workflow could be that you check the `project` page of work that must be done to release a new version. In this case `1.3.2` and maybe these two issues #71 and #72. When all is done then you check that the version / changelog is according to spec and can release a new version.

Reply:
Yeah, if we do not track it in some way, something will definitely will be left out of the change log.

One GitHub feature we could use is Milestones. We just make a milestone (let's say 1.3.2) and add any feature/fix we would like to include in that release under the milestone. Then once all of the issues are resolved under it a version bump can be made from all of the issues and PR assigned to that milestone. What do you think?

Reply:
I removed the version bump from this pull-request and created a milestone for version 1.3.2 that consist of this PR and for #72